server/controllers/students/List.js

```diff
@@ -1,23 +1,50 @@
 const Student = require('../../models/Student')
 const errors = require('../../../shared/errors')
+const utils = require('../../utils')
 
 module.exports = async function listStudents(ctx) {
-  const { course } = ctx.request.query
+  const { course, sort, order } = ctx.request.query
   if (course !== undefined && !['cbcc', 'cbsi'].includes(course)) {
     ctx.status = 400
     ctx.body = { code: errors.INVALID_FILTER, filter: 'course' }
     return
   }
+  if (sort !== undefined && !['name', 'registrationNumber'].includes(sort)) {
+    ctx.status = 400
+    ctx.body = { code: errors.INVALID_SORT_ARGUMENT, filter: 'sort' }
+    return
+  }
+  if (order !== undefined && !['asc', 'desc'].includes(order)) {
+    ctx.status = 400
+    ctx.body = { code: errors.INVALID_ORDER_ARGUMENT, filter: 'order' }
+    return
+  }
   ctx.body = await filterStudents(ctx.request.query)
+  utils.paginateContext(ctx, await filterStudents(ctx.request.query))
 }
 /**
  * Filter students 👍
  * @param {object} query - Filter args
  * @return {Promise} Bookshelf
  */
 function filterStudents(filters) {
-  const { page = 1, name, registrationNumber, minCrg, maxCrg, eqCrg } = filters
+  const {
+    page = 1,
+    name,
+    registrationNumber,
+    minCrg,
+    maxCrg,
+    eqCrg,
+    sort,
+    order
+  } = filters
   let query = Student
+  if (sort !== undefined && order === undefined) {
+    query = query.forge().orderBy(sort, 'ASC')
+  }
+  if (sort !== undefined && order !== undefined) {
```

Comment:
Poderia verificar apenas `sort !== undefined` e usar um valor padrão para o `order`, como sugerido na linha 6. Assim, não há necessidade de usar o `order` quando o `sort` não for enviado bem como evita erros de falta de parâmetro.